x-pack/plugins/security_solution/public/common/components/exceptions/edit_exception_modal/translations.ts

```diff
@@ -67,3 +67,18 @@ export const EXCEPTION_BUILDER_INFO = i18n.translate(
     defaultMessage: "Alerts are generated when the rule's conditions are met, except when:",
   }
 );
+
+export const VERSION_CONFLICT_ERROR_TITLE = i18n.translate(
+  'xpack.securitySolution.exceptions.editException.versionConflictTitle',
+  {
+    defaultMessage: 'Sorry, there was an error',
+  }
+);
+
+export const VERSION_CONFLICT_ERROR_DESCRIPTION = i18n.translate(
+  'xpack.securitySolution.exceptions.editException.versionConflictDescription',
+  {
+    defaultMessage:
+      "It appears this item was updated since you first selected to edit it. Selecting 'Cancel' and re-entering the modal should remove this error.",
```

Comment:
The conflict detection works really well when the users don't edit with comments 😀

@dontcallmesherryli @marrasherrier What do you think the text should say here. I think we should replace the word `item` with `exception` and remove mention of a modal. So maybe something like:

```
It appears this exception was updated since you first selected to edit it. Try clicking 'Cancel' and editing the exception again.
```

Reply:
Updated text!